cli/src/main.rs

```diff
@@ -458,6 +458,8 @@ fn main() {
                 ProofFormat::JWT => {
                     let mut jwt = String::new();
                     credential_reader.read_to_string(&mut jwt).unwrap();
+                    // Trim off whitespace that may have been added by a text editor.
+                    jwt = jwt.trim().to_string();
```

Comment:
As mentioned in <https://github.com/spruceid/didkit/pull/218#issuecomment-921935571>:

> Would it address your usability concern if the trim was done only if standard input is a TTY (i.e. for interactive command-line use, rather than reading directly from a file or pipe)?

Using [atty](https://crates.io/crates/atty):
```suggestion
                    if atty::is(Stream::Stdin) {
                        // Trim off whitespace that may have been added by a text editor.
                        jwt = jwt.trim().to_string();
                    }
```